topi/python/topi/cuda/tensor_intrin.py

```diff
@@ -0,0 +1,54 @@
+"""Tensor intrinsics on CUDA."""
+#pylint: disable=invalid-name
+import tvm
```

Comment:
try use relative import, to get the function you need

Reply:
@tqchen actually we can't, we are in topi, a different package from tvm